tutorials/feature-use/Migration_from_Prophet.ipynb

```diff
@@ -0,0 +1,293 @@
+    "        return 1\n",
+    "    else:\n",
+    "        return 0\n",
+    "df_nfl = df.copy()\n",
+    "df_nfl['nfl_sunday'] = df_nfl['ds'].apply(nfl_sunday)\n",
+    "\n",
+    "# Prophet\n",
+    "p = Prophet()\n",
+    "p = p.add_regressor('nfl_sunday')\n",
+    "p.fit(df_nfl)\n",
+    "future_p = p.make_future_dataframe(periods=30)\n",
+    "future_p['nfl_sunday'] = future_p['ds'].apply(nfl_sunday)\n",
+    "_ = p.predict(future_p)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "np = NeuralProphet()\n",
+    "future_np = np.make_future_dataframe(df_nfl, periods=30)\n",
+    "np = np.add_future_regressor('nfl_sunday')\n",
+    "np.fit(df_nfl)\n",
+    "future_np['nfl_sunday'] = future_np['ds'].apply(nfl_sunday)\n",
+    "_ = np.predict(future_np)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Holidays & Events\n",
+    "\n",
+    "What is referred to as \"holidays\" in Prophet is named \"events\" more generically in NeuralProphet. In Prophet, holidays are passed during the init of the Prophet forecaster. In NeuralProphet, they are added using the `add_events(...)` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prophet\n",
+    "superbowl_p = pd.DataFrame({\n",
+    "  'holiday': 'superbowl',\n",
+    "  'ds': pd.to_datetime(['2010-02-07', '2014-02-02', '2016-02-07']),\n",
+    "  'lower_window': 0,\n",
+    "  'upper_window': 1,\n",
+    "})\n",
+    "\n",
+    "p = Prophet(holidays=superbowl_p)\n",
+    "p = p.fit(df)\n",
+    "future_p = p.make_future_dataframe(periods=30)\n",
+    "forecast_p = p.predict(future_p)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "superbowl_np = pd.DataFrame({\n",
+    "  'event': 'superbowl',\n",
+    "  'ds': pd.to_datetime(['2010-02-07', '2014-02-02', '2016-02-07']),\n",
+    "})\n",
+    "\n",
+    "np = NeuralProphet()\n",
+    "np = np.add_events(\"superbowl\", lower_window=0, upper_window=1)\n",
+    "history_df = np.create_df_with_events(df, superbowl_np)\n",
+    "_ = np.fit(history_df)\n",
+    "future_np = np.make_future_dataframe(history_df, events_df=superbowl_np, periods=30)\n",
+    "forecast_np = np.predict(future_np)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Regularization\n",
+    "\n",
+    "In Prophet, the argument `prior_scale` can be used to configure regularization. In TorchProphet, regularization is controlled via the `reg` argument. `prior_scale` and `reg` have an inverse relationship and therefore cannot directly be translated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prophet\n",
+    "from prophet import Prophet\n",
+    "p = Prophet(seasonality_prior_scale=0.5)\n",
+    "\n",
+    "# NeuralProphet\n",
+    "from neuralprophet import NeuralProphet\n",
+    "np = NeuralProphet(seasonality_reg=0.5)"
```

Comment:
maybe use a different value to make your point about different scales clearer. e.g. 0.1